cmd/aggregated-apiserver/app/options/options.go

```diff
@@ -124,6 +129,7 @@ func (o *Options) Config() (*aggregatedapiserver.Config, error) {
 	}
 
 	serverConfig := genericapiserver.NewRecommendedConfig(aggregatedapiserver.Codecs)
+	serverConfig.LongRunningFunc = customLongRunningRequestCheck(sets.NewString("watch", "follow"), sets.NewString())
```

Comment:
what is `follow `?


why not set subresources?

Reply:
serverConfig.LongRunningFunc = filters.BasicLongRunningRequestCheck(
		sets.NewString("watch", "proxy"),
		sets.NewString("attach", "exec", "proxy", "log", "portforward"),
	)

Reply:
Updated, please take a look.